projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-ai.php

```diff
@@ -45,6 +42,38 @@ public function __construct() {
 		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
 	}
 
+	/**
+	 * Checks if a given request is allowed to get AI data from WordPress.com.
+	 *
+	 * @return true|WP_Error True if the request has access, WP_Error object otherwise.
+	 */
+	public function permissions_check() {
+
+		if ( ! current_user_can( 'edit_posts' ) ) {
```

Comment:
Assuming we agree to make the request to WPCOM using `wpcom_json_api_request_as_blog` this wouldn't work as in such cases (blog token used) we'd get the `rest_forbidden` error

Reply:
Yes, that is why we are not `wpcom_json_api_request_as_blog `